src/source/push.ts

```diff
@@ -284,13 +282,13 @@ export class PushSource {
     return axiosRequestHeaders(this.apikey);
   }
 
+  /**
```

Comment:
This is an error on my side. This method was never meant to be public at the first place.
If the user wants to detect/create missing fields, he should use the FieldAnalyser class